base/broadcast.jl

```diff
@@ -815,6 +816,8 @@ preprocess(dest, x) = extrude(broadcast_unalias(dest, x))
 preprocess_args(dest, args::Tuple{Any}) = (preprocess(dest, args[1]),)
 preprocess_args(dest, args::Tuple{}) = ()
 
+is_simd_safe(bc::Broadcasted) = all(is_simd_safe.(bc.args))
```

Comment:
`all(is_simd_safe, bc.args)`

Reply:
This definition doesn't make sense to me. Either we should assume that `broadcast` is a read-only operation (and this function thus would always be `true`), or we can't assume that about the function, in which case we must always say `false`.